dts/bindings/mtd/mspi-nor-mx25.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024, STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: NOR Flash devices MX25xx on MSPI bus supporting the JEDEC CFI interface
+
+compatible: "mspi-nor-mx25"
```

Comment:
compatible should not have names of subsystems, why not something like `jedec,mx25`

Reply:
changing to jedec,mx25 is fine for me. Then using the DT_HAS_JEDEC_MX25_ENABLED

Should then, the CONFIG_FLASH_MSPI_NOR_MX also be named CONFIG_FLASH_MSPI_JEDEC_MX25 or so ?

Reply:
the kconfig name doesn't matter to me

Reply:
My suggestion is 
```suggestion
compatible: "jedec,mspi-nor-mx25"
```

Reply:
which one do you choose